pkgs/development/python-modules/scenedetect/default.nix

```diff
@@ -0,0 +1,72 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  av,
+  click,
+  numpy,
+  pytestCheckHook,
+  opencv-python,
+  platformdirs,
+  tqdm,
+}:
+let
+  testsResources = fetchFromGitHub {
+    owner = "Breakthrough";
+    repo = "PySceneDetect";
+    rev = "resources";
+    hash = "sha256-7ws7F7CkEJAa0PgfMEOwnpF4Xl2BQCn9+qFQb5MMlZ0=";
+  };
+in
+buildPythonPackage rec {
+  pname = "scenedetect";
+  version = "0.6.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Breakthrough";
+    repo = "PySceneDetect";
+    tag = "v${version}-release";
+    hash = "sha256-G5NLk6eOpclfrzzHad2KT3uZqydSJU0oF/4L2NIdZe0=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [
+    av
+    click
+    numpy
+    opencv-python
+    platformdirs
+    tqdm
+  ];
+
+  pythonImportsCheck = [
+    "scenedetect"
+  ];
+
+  preCheck = ''
+    ln -s ${testsResources}/tests/resources tests/
+  '';
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ];
+
+  disabledTests = [
+    # require write permission on `tests/resources/testvideo.mp4`
+    "test_api"
+    "test_backwards_compat"
+  ];
+
+  meta = {
+    description = "Python and OpenCV-based scene cut/transition detection program & library";
+    homepage = "https://www.scenedetect.com";
+    changelog = "https://github.com/Breakthrough/PySceneDetect/releases/tag/v${version}-release";
+    mainProgram = "scenedetect";
+    sourceProvenance = [ lib.sourceTypes.fromSource ];
```

Comment:
```suggestion
```

I don't think this is needed here

Reply:
Why wouldn't it be? It a good indication whether it comes from pypi, a standalone wheel or from source code.

Reply:
Afaik building from source is already the default in nixpkgs, so I assume this only needs to be specified if you deviate from that. Also, looking at other packages, specifiying `sourceProvenance = [ fromSource ]` explicitly doesn't seem to be too common. Anyway, I definitely wouldn't consider this a blocker, so feel free to just mark this as resolved.

Reply:
You're right. I've check the [relevant section](https://nixos.org/manual/nixpkgs/stable/#sec-meta-sourceProvenance) for the `sourceProvenance`. As both the resources branch and the package itself come from source, it is the default. I've removed it. Good catch!